homeassistant/components/nuki/lock.py

```diff
@@ -1,54 +1,66 @@
-"""Nuki.io lock platform."""
+"""
+Nuki.io lock platform.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/lock.nuki/
+"""
 from datetime import timedelta
 import logging
+import requests
 
 import voluptuous as vol
 
-from homeassistant.components.lock import DOMAIN, PLATFORM_SCHEMA, LockDevice
+from homeassistant.components.lock import (
+    DOMAIN, PLATFORM_SCHEMA, LockDevice, SUPPORT_OPEN)
 from homeassistant.const import (
-    ATTR_ENTITY_ID, CONF_HOST, CONF_PORT, CONF_TOKEN)
+    ATTR_ENTITY_ID, CONF_HOST, CONF_PORT, CONF_TOKEN, CONF_TIMEOUT)
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.service import extract_entity_ids
 
-REQUIREMENTS = ['pynuki==1.3.2']
-
 _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_PORT = 8080
+DEFAULT_TIMEOUT = 5
 
 ATTR_BATTERY_CRITICAL = 'battery_critical'
 ATTR_NUKI_ID = 'nuki_id'
 ATTR_UNLATCH = 'unlatch'
+ATTR_LOCK_REACHABLE = "lock_reachable"
+CONF_UNLATCH_TO_UNLOCK = "unlatch_to_unlock"
 
 MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(seconds=5)
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=30)
 
 NUKI_DATA = 'nuki'
 
 SERVICE_LOCK_N_GO = 'nuki_lock_n_go'
-SERVICE_UNLATCH = 'nuki_unlatch'
+SERVICE_CHECK_CONNECTION = "nuki_check_connection"
```

Comment:
We should no longer register services under the lock domain, instead register under the `nuki` domain.

Reply:
How is it done? Should I change 
from homeassistant.components.lock import (
    DOMAIN, PLATFORM_SCHEMA, LockDevice, SUPPORT_OPEN)
to
from homeassistant.components.nuki import (
    DOMAIN, PLATFORM_SCHEMA, LockDevice, SUPPORT_OPEN)
?

Reply:
hass.services.async_register('nuki', SERVICE_X, …)